pkg/agent/cache/manager.go

```diff
@@ -152,9 +169,14 @@ func NewAgentCacheManager(cfg config.CentralConfig, persistCacheEnabled bool) Ma
 		isCacheUpdated:          false,
 		logger:                  logger,
 		isPersistedCacheEnabled: persistCacheEnabled,
+		migrators:               []cacheMigrate{},
 	}
 
 	if m.isPersistedCacheEnabled {
+		m.migrators = []cacheMigrate{
```

Comment:
Do the other migrations need to be added here?

Reply:
I thought about that, but for things like updating api server resources it may be best for us to just invalidate the cache. These migrators are for gathering info from data already in the cache.